modules/minimp3/resource_importer_mp3.h

```diff
@@ -55,7 +58,9 @@ class ResourceImporterMP3 : public ResourceImporter {
 	virtual bool has_advanced_options() const override;
 	virtual void show_advanced_options(const String &p_path) override;
 #endif
-	static Ref<AudioStreamMP3> import_mp3(const String &p_path);
+
+	static Ref<AudioStreamMP3> load_from_file(const String &p_path);
+	static Ref<AudioStreamMP3> load_from_buffer(const Vector<uint8_t> &p_file_data);
```

Comment:
I would add `Error *r_error` to each function, and create helper `_load_from_file` and `_load_from_buffer` private methods for `_bind_methods` ClassDB registration.

Reply:
These helper functions will have or won't have error return. I honestly don;t really understand reason to have half of the `_methods`

Reply:
See how `FileAccess` and `DirAccess` work with `open()`, to know how they handle exposing the error to the user.

Reply:
TIL that relationship between `method` and `_method` can be not `method` -> `_method` but `_method` -> `method` which is fully opposite to what I expected to see  :exploding_head: What is more mindlowing that I finally understood why sometimes I saw how `_methods` were binded to a name of `method`. Thank you!

Reply:
It depends, sometimes. Haha. It's not 100% set in stone, it depends class by class, honestly.